### PR TITLE
Update Electrum-VTC for the Lyra2REv3 hard fork

### PR DESCRIPTION
**Pull request by metalicjames**
This PR is the counterpart for the hard fork PR on the Vertcoin Core repo: https://github.com/vertcoin-project/vertcoin-core/pull/72

### CODE REVIEW

File: lib/blockchain.py [hunk 1]
```diff
@@ -433,6 +435,9 @@ def get_target(self, height, chain={}):
             bitsN = (bits >> 24) & 0xff
```
Comment (gertjaap) [inline lib/blockchain.py:435]: Indentation seems off here, maybe just github's UI

**Review by gertjaap — APPROVED** [review]
This looks good.

File: lib/blockchain.py (hunk 1 above)
Comment (metalicjames) [reply lib/blockchain.py:435]: The indentation was incorrect in the previous version. But KGW is not used on VTC down to block 0 so the code never ran anyway. This time we have two blocks that are "zero" (one at 0, the other at 1080000) so the indentation is important. The check needs to occur irrespective of whether we have scanned the minimum number of blocks.

**Comment by metalicjames** [comment]
Merged into 3.3.0.

**Pull request closed without merging**